libraries/browser-client/src/websocket/worker.ts

```diff
@@ -321,9 +324,8 @@ class WebsocketClient {
   }
 
   send(data: string | Int16Array): void {
-    if (!this.websocket) {
-      throw new Error('No Websocket')
-    }
+    // We're offline, most likely due to an earlier error
+    if (!this.websocket) return
```

Comment:
Is this now a place where a problem might be hidden? So if the `websocket` is falsy, nobody will know? Would it make sense to throw an Error from here, which might make the whole stack to collapse and that would expose the potential problem?